platformio.ini

```diff
@@ -47,6 +47,13 @@ build_flags =
 	-DFW_VERSION="DEBUG-ETH"
 monitor_speed = 115200
 
+[env:room8266-debug-wifi]
+extends = room8266-wifi
```

Comment:
Not sure this will work as I think it will bring in the release script?

In other projects I have just extended the base `room8266` env and added the WiFi dependencies and flags in the debug section as well, effectively duplicating the WiFi release env.

Reply:
I tried it out and the room286-debug-wifi worked as expected (compiled a binary which was flashed and acessible).
It was my goal to avoid duplicated sections.

Reply:
I think one of the biggest builds is the led firmware, I think the way we setup the parameters there worked / works well 
[https://github.com/austinscreations/OXRS-AC-LEDController-ESP-FW](url)

Reply:
I think the 2 platformio.ini files follow different concepts. 
The `LED-Controller` has all dependencies collected in the global `[env]` section whereas the `StateMonitor `seperates the dependencies depending on the FW to be built.
Don'tknow what the better path forward is. Up to @sumnerboy12

Reply:
I always assumed we could re-use the release sections as they included the release script. But it sounds like that was an incorrect assumption. I would much prefer Moin's approach - a single definition for libs and build args, shared for release and debug builds.